<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FitDlg</class>
 <widget class="QDialog" name="FitDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>449</width>
    <height>529</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Fit</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabs">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabUser">
      <attribute name="title">
       <string>User-defined Fit</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="1">
        <widget class="QComboBox" name="comboFitType">
         <item>
          <property name="text">
           <string>1D Fit</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>2D Fit</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Function:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="editFkt"/>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Type:</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0" colspan="2">
        <widget class="QTableWidget" name="tableHints">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="verticalScrollMode">
          <enum>QAbstractItemView::ScrollPerItem</enum>
         </property>
         <property name="horizontalScrollMode">
          <enum>QAbstractItemView::ScrollPerItem</enum>
         </property>
         <attribute name="horizontalHeaderDefaultSectionSize">
          <number>128</number>
         </attribute>
         <column>
          <property name="text">
           <string>Hint</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Deviation</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Active</string>
          </property>
         </column>
        </widget>
       </item>
       <item row="5" column="0" colspan="2">
        <widget class="QTableWidget" name="tableLimits">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="verticalScrollMode">
          <enum>QAbstractItemView::ScrollPerItem</enum>
         </property>
         <property name="horizontalScrollMode">
          <enum>QAbstractItemView::ScrollPerItem</enum>
         </property>
         <property name="gridStyle">
          <enum>Qt::SolidLine</enum>
         </property>
         <attribute name="horizontalHeaderDefaultSectionSize">
          <number>128</number>
         </attribute>
         <attribute name="horizontalHeaderMinimumSectionSize">
          <number>23</number>
         </attribute>
         <column>
          <property name="text">
           <string>Lower limit</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Upper limit</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Active</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabSpecial">
      <attribute name="title">
       <string>Special Fit</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="7" column="1">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="6" column="2">
        <widget class="QSpinBox" name="spinFoil">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="minimum">
          <number>-1</number>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QRadioButton" name="radio1D">
         <property name="text">
          <string>1D Fit</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="5" column="1" colspan="2">
        <widget class="QCheckBox" name="checkOnlyContrast">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Only create contrast graph</string>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QLabel" name="labelFoil">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Foil (for TOF data):</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QRadioButton" name="radio1DPixel">
         <property name="text">
          <string>2D Fit (pixel-wise)</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="labelPixel">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Function:</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1" colspan="2">
        <widget class="QComboBox" name="comboBoxSpecialFktPixel">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <item>
          <property name="text">
           <string>MIEZE Sine</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>MIEZE Sine (FFT)</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="labelFkt">
         <property name="text">
          <string>Function:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1" colspan="2">
        <widget class="QComboBox" name="comboSpecialFkt">
         <item>
          <property name="text">
           <string>MIEZE Sine</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Gaussian</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Multi-Gaussian</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QSpinBox" name="spinParam">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="value">
          <number>2</number>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLabel" name="labelParam">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Parameter:</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Data Sources</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0" colspan="3">
       <widget class="QListWidget" name="listGraphs">
        <property name="dragEnabled">
         <bool>true</bool>
        </property>
        <property name="dragDropMode">
         <enum>QAbstractItemView::InternalMove</enum>
        </property>
        <property name="defaultDropAction">
         <enum>Qt::IgnoreAction</enum>
        </property>
        <property name="alternatingRowColors">
         <bool>true</bool>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::ExtendedSelection</enum>
        </property>
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectItems</enum>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="btnAdd">
        <property name="text">
         <string>Add Items...</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="btnAddActive">
        <property name="text">
         <string>Add Active Item</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QPushButton" name="btnDel">
        <property name="text">
         <string>Remove Items</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame_2">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="lineWidth">
      <number>0</number>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="margin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="labelStatus">
        <property name="text">
         <string>Ready.</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDialogButtonBox" name="buttonBox">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="standardButtons">
         <set>QDialogButtonBox::Apply|QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>comboFitType</tabstop>
  <tabstop>editFkt</tabstop>
  <tabstop>tableLimits</tabstop>
  <tabstop>tableHints</tabstop>
  <tabstop>radio1D</tabstop>
  <tabstop>comboSpecialFkt</tabstop>
  <tabstop>spinParam</tabstop>
  <tabstop>radio1DPixel</tabstop>
  <tabstop>comboBoxSpecialFktPixel</tabstop>
  <tabstop>checkOnlyContrast</tabstop>
  <tabstop>spinFoil</tabstop>
  <tabstop>listGraphs</tabstop>
  <tabstop>btnAdd</tabstop>
  <tabstop>btnAddActive</tabstop>
  <tabstop>btnDel</tabstop>
  <tabstop>buttonBox</tabstop>
  <tabstop>tabs</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>FitDlg</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>68</x>
     <y>457</y>
    </hint>
    <hint type="destinationlabel">
     <x>218</x>
     <y>232</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>FitDlg</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>68</x>
     <y>457</y>
    </hint>
    <hint type="destinationlabel">
     <x>218</x>
     <y>232</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radio1DPixel</sender>
   <signal>toggled(bool)</signal>
   <receiver>comboBoxSpecialFktPixel</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>82</x>
     <y>99</y>
    </hint>
    <hint type="destinationlabel">
     <x>290</x>
     <y>124</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radio1DPixel</sender>
   <signal>toggled(bool)</signal>
   <receiver>labelPixel</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>82</x>
     <y>99</y>
    </hint>
    <hint type="destinationlabel">
     <x>44</x>
     <y>124</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radio1D</sender>
   <signal>toggled(bool)</signal>
   <receiver>labelFkt</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>48</x>
     <y>49</y>
    </hint>
    <hint type="destinationlabel">
     <x>44</x>
     <y>74</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radio1D</sender>
   <signal>toggled(bool)</signal>
   <receiver>comboSpecialFkt</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>48</x>
     <y>49</y>
    </hint>
    <hint type="destinationlabel">
     <x>290</x>
     <y>74</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radio1DPixel</sender>
   <signal>toggled(bool)</signal>
   <receiver>checkOnlyContrast</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>82</x>
     <y>99</y>
    </hint>
    <hint type="destinationlabel">
     <x>290</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radio1DPixel</sender>
   <signal>toggled(bool)</signal>
   <receiver>labelFoil</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>85</x>
     <y>99</y>
    </hint>
    <hint type="destinationlabel">
     <x>224</x>
     <y>174</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radio1DPixel</sender>
   <signal>toggled(bool)</signal>
   <receiver>spinFoil</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>85</x>
     <y>99</y>
    </hint>
    <hint type="destinationlabel">
     <x>362</x>
     <y>174</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radio1DPixel</sender>
   <signal>toggled(bool)</signal>
   <receiver>labelParam</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>85</x>
     <y>125</y>
    </hint>
    <hint type="destinationlabel">
     <x>224</x>
     <y>100</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radio1DPixel</sender>
   <signal>toggled(bool)</signal>
   <receiver>spinParam</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>85</x>
     <y>125</y>
    </hint>
    <hint type="destinationlabel">
     <x>362</x>
     <y>100</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
